embed/extmod/modtrezorcrypto/modtrezorcrypto-bip32.h

```diff
@@ -300,6 +338,20 @@ STATIC mp_obj_t mod_trezorcrypto_HDNode_public_key(mp_obj_t self) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(mod_trezorcrypto_HDNode_public_key_obj, mod_trezorcrypto_HDNode_public_key);
 
+/// def public_key_cardano(self) -> bytes:
+///     '''
+///     Returns public key without the first bytes of the HD node.
+///     '''
+STATIC mp_obj_t mod_trezorcrypto_HDNode_public_key_cardano(mp_obj_t self) {
```

Comment:
Don't introduce new function. Just call `public_key` and strip the first byte using `[1:]` in python.